lib/_http_outgoing.js

```diff
@@ -65,7 +65,7 @@ function isCookieField(s) {
 
 function noopPendingOutput(amount) {}
 
-function OutgoingMessage() {
+function OutgoingMessage( ) {
```

Comment:
format: style mismatch:
```suggestion
function OutgoingMessage() {
```

`cc749a7d` ␣ at column 26 should be removed.